### PR TITLE
Fix issue #6 - Change priority of keys to env then file

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@brianloveswords I think we should release a patch version that includes the fixes I did previously, then merge this in and release it was v2.0 since it's a breaking change

File: README.md [hunk 1]
```diff
@@ -108,7 +108,13 @@ var nodeEnv = habitat.get('nodeEnv');
 ```
 
 ## habitat.load([*pathToEnvFile*])
-Try to load a set of environment variables from a file. This **will** override whatever is in the environment.
+Try to load a set of environment variables from a file. This **will not** override whatever is in the environment. This is a change from Habitat v1.x's behaviour. This means that you can chain multiple environment files together to provides sane defaults for your local development or to commit environment configuration into your repository:
+
+```js
+habitat.load('.env');
```
Comment (humphd) [inline README.md:118]: What if `load` can take a String (1.x behaviour, single file) or an Array of String (2.x behaviour, with overrides happening based on ordering)?

Comment (brianloveswords) [inline README.md:118]: I think my preference would be this to take a second, optional argument object which can determine whether the behavior is to fill-in or override.

``` js
// Precedence:
//   system environment
//   config/production.env
//   config/default.env
habitat.load([
  "config/production.env",
  "config/default.env"
], { override: false });
```

``` js
// Precedence:
//   config/production.env
//   config/default.env
//   system environment
habitat.load([
  "config/default.env"
  "config/production.env",
], { override: true });
```

Depending on whether the default for `override` is `true` or `false` would determine whether we have to bump the major version.

File: README.md [hunk 2]
```diff
@@ -108,7 +108,13 @@ var nodeEnv = habitat.get('nodeEnv');
 ```
 
 ## habitat.load([*pathToEnvFile*])
-Try to load a set of environment variables from a file. This **will** override whatever is in the environment.
+Try to load a set of environment variables from a file. This **will not** override whatever is in the environment. This is a change from Habitat v1.x's behaviour. This means that you can chain multiple environment files together to provides sane defaults for your local development or to commit environment configuration into your repository:
```
Comment (sedge) [inline README.md]: "...to **provides** sane defaults..." -> "...to **provide**..."

**Pull request merged**